src/core/credentials/transport/google_default/google_default_credentials.h

```diff
@@ -48,6 +48,13 @@
   "/" GRPC_GOOGLE_WELL_KNOWN_CREDENTIALS_FILE
 #endif
 
+enum class DefaultCallCredentialsCreationMethod {
```

Comment:
This should be moved to the .cc file, since it is no longer needed external to this file.